app/controllers/application_controller.rb

```diff
@@ -5,7 +5,7 @@ class ApplicationController < ActionController::Base
   # before_action :ensure_logged_in
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  # before_action :ensure_logged_in
+  before_action :ensure_logged_in
```

Comment:
This will restrict us from having public pages in the system. 

An example on how we manage this in each controller could be seen [here](https://github.com/pupilfirst/pupilfirst/blob/0fb953e2626acecde28885cccab48a6c47a417fb/app/controllers/courses_controller.rb#L20).

@ProCode2 could you create an issue to migrate this issue.

Reply:
Oh, Okay will create an issue asap.